rebasehelper/specfile.py

```diff
@@ -195,19 +195,21 @@ def guess_category():
     # TAG HELPER METHODS #
     ######################
 
-    def tag(self, name: str, section: Optional[str] = None) -> Optional[Tag]:
+    def tag(self, name: str, section: Optional[str] = None, section_index: Optional[int] = None) -> Optional[Tag]:
```

Comment:
This is a convenience method returning the first matching tag, so it doesn't make much sense to make it more complex by allowing to specify a particular section. If you want to do that anyway, I think the `section` argument should accept both name (`str`) and index (`int`). But `get_raw_tag_value()` and `set_raw_tag_value()` probably shouldn't use this method at all.